src/sdk/conference/channel.js

```diff
@@ -79,7 +89,7 @@ export class ConferencePeerConnectionChannel extends EventDispatcher {
     }
   }
 
-  publish(stream, options, videoCodecs) {
+  async publish(stream, options, videoCodecs) {
```

Comment:
We may throw exception instead of `return Promise.reject` since this method is changed to an async function. This can be done in a separate change. Please make sure the toolchain works for `async`/`await`.